process/consensus.md

```diff
@@ -59,7 +60,7 @@ official video calls. In the latter case, the consensus vote must be added to
 the agenda at least 24 hours before the video call is scheduled to begin, except
 in the case of general interest votes moving pre-proposals to phase 1, which can
 be added as the discussion proceeds. Consensus decisions are recorded in meeting
-notes and published just like in-person meeting notes are published.
+notes and published just like meeting notes are published.
```

Comment:
Maybe just "Consensus decisions are recorded in the published meeting notes"

Reply:
Fixed, thanks!